airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceSSLCertificateAcceptanceTest.java

```diff
@@ -57,8 +57,8 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
         .put("port", container.getFirstMappedPort())
         .put("database", container.getDatabaseName())
         .put("schemas", Jsons.jsonNode(List.of("public")))
-        .put("username", container.getUsername())
```

Comment:
Why don't we use container creds anymore?

Reply:
because I have configured an SSL connection for the postgres user that I create when starting the container. The Client certificate was created specifically for this user, which I use for tests.